grails-spring/src/main/groovy/org/grails/spring/DefaultBeanConfiguration.java

```diff
@@ -230,6 +237,19 @@ protected AbstractBeanDefinition createBeanDefinition() {
         return bd;
     }
 
+    private boolean hasScheduledMethod(Class clazz) {

```

Comment:
I created these methods to clean things up, but feel like they don't fit in this class. Should these go somewhere else?